tools/package-windows.bat

```diff
@@ -33,6 +35,8 @@ cmake -G "Visual Studio 16 2019" -Thost=x64 -A x64 ^
       -DCMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%/scripts/buildsystems/vcpkg.cmake ^
       -DWITH_TESTS=NO -DWITH_APPS=NO -DWITH_TUTORIALS=NO ^
       -DWITH_DOCS=YES -DWITH_UTILS=NO -DWITH_PYTHON_BINDINGS=NO ^
+      "-DCMAKE_INSTALL_BINDIR=bin/$<CONFIG>" "-DCMAKE_INSTALL_LIBDIR=lib/$<CONFIG>" ^
+      "-DHALIDE_INSTALL_CMAKEDIR=lib" ^
       -S . -B build/static
 
 if %errorlevel% neq 0 goto return
```

Comment:
`REM LLVM Debug + Halide exceeds the COFF 4GB limit!!`

as you know, I'm fine eliding the debug builds entirely, but buildbot now gets around this by using `/bigobj` for all Windows Debug LLVM builds. Can we just do that here too?

Reply:
No, because this is an issue with the static library itself, not with any individual object file.